### DISCUSSION TITLE
Trying nbb with a @google-cloud module

### DISCUSSION THREAD

**Discussion by slimslenderslacks** [babashka/nbb#165] [Show and tell]
This worked really well.  I tried out the nlp sentiment module running from nbb

https://slimslenderslacks.github.io/nbb-sentiment

Part way through, I wondered whether `babashka.process` might be available but I think that's still only available for bb scripting.  It was nice to see how easy it is to use an npm module though.  Learning curve was neglible.

**Comment by borkdude** [comment]
@slimslenderslacks Cool! Yes, babashka.process is JVM only. For node there are several libraries available, e.g. the built-in child_process or the (external) execa library.

**Reply by slimslenderslacks** [reply]
Thanks!  That makes sense.

**Reply by borkdude** [reply]
Nice blog btw. I shared it on Clojurians Slack and Twitter. I'll post a link on Reddit too.

**Reply by slimslenderslacks** [reply]
Ya, sounds good.  I'll also write up a completely bizarre scenario where I used nbb to script against an npm module which had itself been compiled from clojurescript (via shadow-cljs).  This can't possibly be a good idea. However, the existing clojurescript module used core.async so I just kind of wondered if it would even work.  And it did.  So I ended up with shadow-cljs watching a clojurescript codebase that exported async functions, being "npm linked" to an nbb project which was running those javascript functions.  What???  But ya, that does actually work.

**Reply by borkdude** [reply]
Hahaha, amazing!
